ada/extensions/playground

```diff
@@ -0,0 +1,29 @@
+## vim: filetype=makopython
+
+    def create_arg_parser(self):
+        Playground.create_arg_parser(self)
+        self.parser.add_argument(
+            '-X', action='append',
+            help="Scenario variables to pass along to GPR"
+        )
+        self.parser.add_argument(
+            '-P', '--project', type=str, default='', help="GPR project file"
+        )
+
+    def process_args(self):
+        from tempfile import mkdtemp
+        if not self.args.project:
+            with open("{}/default.gpr".format(mkdtemp()), 'w') as f:
+                f.write("project Default is end Default;")
+                self.args.project = f.name
```

Comment:
If I understand correctly, this will clutter /tmp without ever cleaning it up. Not cool!